graphql-server/pkg/queries/reaction.go

```diff
@@ -105,3 +119,24 @@ func (q *TargetReactionQuery) QueryTargetReactions() ([]ReactionCounts, error) {
 
 	return result, nil
 }
+
+func (q *UserReactionQuery) NewQuery() *bun.SelectQuery {
+	return q.session.NewSelect().
+		Model((*models.Reaction)(nil)).
+		Where("address = (?)", q.scopedUserAddress)
+}
+
+func (q *UserReactionQuery) QueryUserReactions() (map[string]models.Reaction, error) {
```

Comment:
you were right to return []models.Reaction here (in order of scoped targets) so we follow the usual pattern

should handle the mapping at loading logic instead